crates/primitives/src/eip4844.rs

```diff
@@ -0,0 +1,24 @@
+//! Helpers for working with EIP-4844 blob fee
+
+use crate::{H256, constants::eip4844::{TARGET_DATA_GAS_PER_BLOCK, VERSIONED_HASH_VERSION_KZG}};
+use c_kzg::KzgCommitment;
```

Comment:
can use `crate::{kzg::KzgCommitment, // rest ... }` since `c_kzg` is re-exported by primitives